### PR TITLE
add missing Howl.seek() function overloads

### PR DESCRIPTION
**Pull request by hirohe**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://github.com/goldfire/howler.js#seekseek-id>

document mentions seek function optionally takes 0, 1 or 2 arguments. but seek function with 1 argument can be confuse, it could be set the position of playback for a sound Or get the position of playback by id, here is [sample](https://codesandbox.io/s/epic-agnesi-j2mjq?file=/src/index.js) of seek function's return type

- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by jrandolf-old — CHANGES_REQUESTED** [review]
This is not how you overload. See https://www.typescriptlang.org/docs/handbook/2/functions.html#overload-signatures-and-the-implementation-signature

File: types/howler/index.d.ts [hunk 1]
```diff
@@ -208,7 +208,9 @@ export class Howl {
     rate(idOrSetRate: number): this | number;
     rate(rate: number, id: number): this;
 
-    seek(seek?: number, id?: number): this | number;
+    seek(): number;
+    seek(idOrSetSeek: number): this | number;
```
Comment (jun-sheaf) [inline types/howler/index.d.ts:212]: What is `idOrSetSeek`? Why does this return `this` or a `number`?  In fact, they should all return a single type, not a union. Or document why it's not for `seek(idOrSetSeek)`.

**Review by jrandolf-old — CHANGES_REQUESTED** [review]
Add tests.

File: types/howler/index.d.ts (hunk 1 above)
Comment (hirohe) [reply types/howler/index.d.ts:212]: here is [document ](https://github.com/goldfire/howler.js#seekseek-id) of the seek function

here is [codesandbox](https://codesandbox.io/s/epic-agnesi-j2mjq?file=/src/index.js) shows the return type of seek function

**Comment by hirohe** [comment]
`<my-package>-tests.ts` test file does not run, only for type-checked.

how can i test the seek function's return type? just add the usage of seek function?

File: types/howler/index.d.ts (hunk 1 above)
Comment (jun-sheaf) [reply types/howler/index.d.ts:212]: The documentation does not describe this. Only the code does. Please comment this.

Also, please add tests.

**Comment by jun-sheaf** [comment]
Yes, just add the usage of the `seek` function.

**Review by orta — APPROVED** [review]
Looks reasonable to me 👍🏻

**Pull request merged**